src/components/ebay-spark-line/style.js

```diff
@@ -0,0 +1 @@
+require('./spark-line.less');
```

Comment:
Why not just put `spark-line.less` contents in this file?

Reply:
@agliga what would the syntax be for this. I don't think I've ever includes less directly in a js file like that.

Reply:
Oh I see, it's a js file. 
Let me check on this.

Reply:
So for now:
lets have a `style.less` with all the styles.
Lets have a `style.js` which imports `style.less`

Reply:
ok so leave it as is then

Reply:
oh wait I see rename the less file. change coming in a minute